src/app/add-case-dialogue/add-case-dialogue.component.html

```diff
@@ -0,0 +1,14 @@
+<h1 mat-dialog-title>Add Positive Case</h1>
+<div mat-dialog-content>
+  <p>What is the date of the person's test?</p>
+  <mat-form-field appearance="fill">
+    <mat-label>Choose a date</mat-label>
+    <input matInput [matDatepicker]="picker" [(ngModel)]="data.testDate">
```

Comment:
Interesting, so this matDatepicker is internal to angular?

Reply:
It comes from Material, so I don't really know if you would consider that internal to angular or not.

Reply:
Nope. It's a wonderful add on!

However most people seem to prefer bootstrap, so you'll see that too.

Though for stringing up a quick app, especially when you don't know angular... angular materials is really good at teaching and being easy to built cool things.